File: task3/helloworld.js
function isPrime(num) {
    if (num < 2) {
        return false;
    }
    for (let i = 2; i <= Math.sqrt(num); i++) {
        if (num % i === 0) {
            return false;
        }
    }
    return true;
}

const readline = require('readline-sync');

let n = parseInt(readline.question("Enter a number: "));

console.log(`Prime numbers up to ${n}:`);
for (let i = 2; i <= n; i++) {
    if (isPrime(i)) {
        console.log(i);
    }
}

File: task3/helloworld.go
package main

import (
	"fmt"
	"math"
)

func isPrime(num int) bool {
	if num < 2 {
		return false
	}
	for i := 2; i <= int(math.Sqrt(float64(num))); i++ {
		if num%i == 0 {
			return false
		}
	}
	return true
}

func main() {
	var n int
	fmt.Print("Enter a number: ")
	fmt.Scan(&n)

	fmt.Printf("Prime numbers up to %d:\n", n)
	for i := 2; i <= n; i++ {
		if isPrime(i) {
			fmt.Println(i)
		}
	}
}
